Project-Days-Calendar/README.md

```diff
@@ -0,0 +1,111 @@
+
+The goal of this project is to present this data usefully to users.
+
+## Requirements
+
+This may be an individual or a team project - your class leaders will tell you which you are doing.
+
+Some of the requirements of this project are only required if you are working in a group of at least a certain size.
+
+You must submit both a link to your GitHub repo, and a link to the deployed website.
+
+Your website must be hosted on the internet, and must be automatically deployed when you merge changes to your GitHub repo.
+
+You must be able to explain every line of code in your project, even ones other people in your group wrote.
+
+### Requirements for everyone
+
+Regardless of your group size, you must:
+
+* Create an HTML page which, when loaded, displays a calendar.
+* The calendar must show every day of the current month, each as a rectangle. Each row of rectangles must show one week. The first column must show Mondays. The first day of the month must be shown in the first row.
+* There must be two links or buttons which, when clicked, switch what is displayed. One button must change the display to the previous month. The other button must change the display to the next month. On repeated clicks, these buttons must keep moving back/forwards in time, one month per click.
+* There must be a way to jump to a particular month and year, e.g. "October 2020".
+* The days from the JSON file must appear correctly when the month they fall in is displayed. For example:
+  * If October 2024 is being shown, October 8th must show Ada Lovelace Day.
+  * If October 2025 is being shown, October 14th must show Ada Lovelace Day.
+* The calendar should work for every year - if someone goes to 1900, or 2050, or any other year, the commemerative days should be correctly displayed.
+* The calendar must be dynamically generated by reading the JSON file. You must not hard-code any days in the HTML file.
+* Other than the above styling requirements, no styling is required.
+
+### Requirements for groups of at least 2
+
+As well as all previous requirements, you must:
+
+* Produce a script that can be run in a terminal via `node`, which outputs to standard out (via calls to `console.log`) an [iCal format](https://icalendar.org/) file containing entries for every day in the JSON file. You must not use recurring events. There must be one entry per commemerative day per year from 2020 until 2030 (inclusive).
+* If you save this output to a file, and import it into Google Calendar, it should show the days correctly in the calendar. There is a suggested workflow for testing this listed below.
+* Logic for calculating dates must be shared between the web generator and the iCal generator.
+
+### Requirements for groups of at least 3
+
+As well as all previous requirements, you must:
+* Fetch a description of the day from the URL supplied in the JSON file.
+* In the HTML page, if you click on a listed commemorative day, the page should show a description of the day.
```

Comment:
Again, I think it might be worth giving some UX guidance here to avoid rabbit-holing. I assume you're thinking of a modal, not a page navigation?